scripts/check_style.sh

```diff
@@ -20,6 +20,11 @@ EXCLUDE_FILES=(
 EXCLUDE_FILES_JOINED=$(printf "%s\|" "${EXCLUDE_FILES[@]}")
 EXCLUDE_FILES_JOINED=${EXCLUDE_FILES_JOINED%??}
 
+INCLUDE_DIRECTORIES=(
```

Comment:
Well, this seems to be specific to the `using namespace std` check so I'd give it a more specific name:
```suggestion
NAMESPACE_STD_FREE_DIRS=(
```